presto-openapi/src/main/resources/query.yaml

```diff
@@ -0,0 +1,122 @@
+openapi: 3.0.0
+info:
+  title: Presto Query Info
+  description: Optional multiline or single-line description in [CommonMark](http://commonmark.org/help/) or HTML.
+  version: 1
+servers:
+  - url: http://api.example.com/v1
+    description: Optional server description, e.g. Main (production) server
```

Comment:
Let's remove this server since there is no "production" server for OSS. It's up to users to deploy their own endpoint.